cmd/mdmctl/config.go

```diff
@@ -8,6 +8,7 @@ import (
 	"net/url"
 	"os"
 	"os/user"
+	"path"
```

Comment:
`path/filepath` has the same methods but looks like it would handle edge cases better.